### ISSUE TITLE
Better Design for Current Status on home page

### ISSUE THREAD

**Issue by DanGiaime** [DanGiaime/ClipbotFrontend#8]
The current status section of the home page is not nearly loud enough or active enough.

Update the css/html/js if needed to make this frontend better and more obvious to the user as it is the most important thing in the app.

**Comment by zainulabideen1122** [comment]
Can I try? Also, give me some detail; what do you want. From the above info, I think you asked to update navigation.

**Comment by DanGiaime** [comment]
> Can I try? Also, give me some detail; what do you want. From the above info, I think you asked to update navigation.

Absolutely! Here's a more detailed description.

![image](https://user-images.githubusercontent.com/21694955/136360118-a9bd8c20-03ad-4ac1-b609-757379c5657c.png)

This is the dashboard right now. The thing we are focus on is the "Current Status" box

![image](https://user-images.githubusercontent.com/21694955/136360172-e7d078a3-5e8d-4c24-a1c0-357db5074cd1.png)

This is the most important box, and there's a bunch of space we could use to make it bigger / better.

I'm looking for someone to come up with a better design for that data in particular, the words "current status" and the actual status (in this case it's "waiting for next upload")

Honestly even just a reasonable centered element with bigger text would already be a big improvement!

Maybe even a slight color change or background to make it more clear how important it is, since we want users to look here.

Hope that clarifies!

**Comment by zainulabideen1122** [comment]
Yeah got it, Can you assign this task to me?

**Comment by DanGiaime** [comment]
Looks awesome! I would like to keep the width though. Maybe something more vertical?
like

          Current Status          
Waiting until next upload...

But I think the green+bold text is perfect

**Comment by zainulabideen1122** [comment]
Something like this?

![clipbot_tv](https://user-images.githubusercontent.com/59528492/136513321-26a4afc6-4c1e-4de5-951f-e303b9912e8d.png)

**Comment by zainulabideen1122** [comment]
@DanGiaime  Like this?
![clipbot_tv](https://user-images.githubusercontent.com/59528492/136534060-7fec3052-86c2-4054-9924-8af173fb3c3f.png)

**Comment by DanGiaime** [comment]
@zainulabideen1122 Yes! That's solid and definitely an improvement. Go ahead and make a PR.

Make sure:
* No element IDs/selectors change
* No width change on the window

And I'll merge :D
